docs/_includes/common/eek.html

```diff
@@ -0,0 +1,242 @@
+<div id="eek">
+    {% include common/section-header.html name="eek" version=page.versions.eek %}
+    <p>
+        EEKs have two parts, a range, and a rating.
+        The following ranges are available:
+    </p>
+    <table cellpadding="0" cellspacing="0" class="icon-table">
+        <thead>
+            <tr>
+                <th style="width:20%;">EEK MIN</th>
+                <th style="width:20%;">EEK MAX</th>
+                <th>ICON</th>
+            </tr>
+        </thead>
+        <tbody>
+            {% assign icons = "A+++-D, A+++-E, A+++-G, A++-E, A++-G, A+-F, A+-E, A-G" |
+            split: ", " %}
+            {% for iconName in icons %}
+
+            {% assign range = iconName | split: "-" %}
+            {% assign min = range | last %}
+            {% assign max = range | first %}
+            <tr>
+                <td>{{ min }}</td>
+                <td>{{ max }}</td>
+                <td>
+                    <span class="demo">
+                        <span class="demo__inner">
+                            {% include common/eek-range.html min=min max=max %}
+                        </span>
+                    </span>
+                </td>
+
+            </tr>
+            {% endfor %}
+        </tbody>
+    </table>
+
+    <p>
+        And for each range, making sure each rating exists within that given range, the EEK can have the following
```

Comment:
Is the end of this sentence correct?

Reply:
Yes, because it leads into the table that follows.

Reply:
Okay, seems a bit odd though to end in a `:` then have a paragraph in between the table.